pyfcm/baseapi.py

```diff
@@ -52,7 +53,7 @@ def __init__(self, api_key=None, proxy_dict=None, env=None, json_encoder=None, a
         self.FCM_REQ_PROXIES = None
         self.requests_session = requests.Session()
         retries = Retry(backoff_factor=1, status_forcelist=[502, 503],
-                        allowed_methods=(Retry.DEFAULT_ALLOWED_METHODS | frozenset(['POST'])))
+                        method_whitelist=(Retry.DEFAULT_METHOD_WHITELIST | frozenset(['POST'])))
```

Comment:
Is this revert intentional?

Reply:
Yes, discovered it started breaking people's apps and had to revert, could be something with the urllib version

Reply:
Yeah from what I see, the arguments were renamed in v1.26.0 https://github.com/urllib3/urllib3/releases/tag/1.26.0
so since then it gives a deprecation warning, since they are going to be removed in v2,  so if you want to use them, you need to change the dependency to minimum v1.26  ,  though if you you want to keep this one for compatibility, you should also add <= 2.0 so that when urllib3 v2 gets released, it wont try to update it